modules/ppcp-api-client/src/Endpoint/class-requesttrait.php

```diff
@@ -36,6 +38,14 @@ private function request( string $url, array $args ) {
 			$args['headers']['PayPal-Partner-Attribution-Id'] = 'Woo_PPCP';
 		}
 
-		return wp_remote_get( $url, $args );
+		$response = wp_remote_get( $url, $args );
+
+		if ( $this->logger instanceof WooCommerceLogger ) {
+			$this->logger->logRequestResponse( $url, $args, $response );
```

Comment:
Would be great to put it somewhere else instead of the logger. Maybe just some trait returning a string.